crimsobot/cogs/image.py

```diff
@@ -227,10 +268,7 @@ async def pingbadge(self, ctx: commands.Context, image: Optional[str] = None) ->
         """Add Discord notification badge to an image."""
 
         if image is None:
-            try:
-                ctx.message.attachments[0].url
```

Comment:
just checking - this means verpingt will no longer accept an attachment? or will it grab it from the message history even when it's an attachment of the invoking message?

Reply:
That was vestigial code. Image fetching has been generalized since then to work for all image commands via `get_image_and_embed()` and the code behind it.